frameworks/TypeScript/bun/bun.dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM oven/bun
```

Comment:
Could you lock this down to a specific version please

Reply:
Done